test.js

```diff
@@ -0,0 +1,157 @@
+'use strict'
+
+const test = require('tap').test
+const sinon = require('sinon')
```

Comment:
I'm not a fan ok mocking. I would really prefer to have at least 1 test against the live system.
However, I'm ok with the change.